python/tvm/auto_scheduler/task_scheduler.py

```diff
@@ -337,10 +338,40 @@ def tune(self, tune_option, search_policy="default"):
             self._tune_task(task_idx)
             self._adjust_similarity_group(task_idx)
 
+    def _print_table_info(self, next_task_idx):
+        # table header
+        _ffi_api.PrintTitle("Task Scheduler")
+        print("|  ID  | Latency (ms) | Speed (GFLOPS) | Trials |")
```

Comment:
Could we extract more information? Like operator name (Conv2D, softmax...) and its shape (1x3x224x224)? Only ID, we have to match its detail information again.

Reply:
It is not easy to extract them by parsing the compute_dag.
One way to achieve this is to attach this information by `attrs` in `te.compute`.
I will leave this to future PRs.

Reply:
Ok